README.md

```diff
@@ -1 +1,77 @@
-# Metrics-webapp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Metrics WebApp
+
+> In this project I built a mobile web application to check a list of countries COVID status making use of React and Redux.
+
+
+## Screenshot
+
+<img src="./App2.png">
+<img src="./App1.png">
+
+## Built With
+
+- React
+- Redux
+- SCSS
+- JavaScript
+- HTML
+
+## Live Demo
+
+For a live demo
+- Netlify [click here](https://alexr16-makes-great-sites.netlify.app/).
```

Comment:
- [ ] Your live demo does not seem to work. Kindly fix it.

  ![alexr-live](https://user-images.githubusercontent.com/36542986/169158291-21cbe586-fa52-4c67-9b2b-aedf199d6324.jpg)